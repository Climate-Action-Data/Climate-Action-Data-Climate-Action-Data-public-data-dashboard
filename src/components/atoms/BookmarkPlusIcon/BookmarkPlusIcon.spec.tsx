import { render } from '@testing-library/react'
import { BookmarkPlusIcon } from './BookmarkPlusIcon'

it(`renders correctly`, () => {
  const { container } = render(<BookmarkPlusIcon />)
  expect(container).toMatchSnapshot()
})
