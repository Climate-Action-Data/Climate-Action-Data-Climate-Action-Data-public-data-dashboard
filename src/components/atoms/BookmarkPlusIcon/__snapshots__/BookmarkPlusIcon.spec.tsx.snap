// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <svg
    class="chakra-icon css-n059si"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M5.76537 20.065C5.46411 20.192 5.1779 20.1678 4.90675 19.9925C4.63558 19.8172 4.5 19.5619 4.5 19.2266V5.66512C4.5 5.15999 4.675 4.73242 5.025 4.38242C5.375 4.03242 5.80257 3.85742 6.3077 3.85742H12V5.3574H6.3077C6.23077 5.3574 6.16024 5.38945 6.09613 5.45355C6.03202 5.51766 5.99997 5.58819 5.99997 5.66512V18.3074L11 16.1574L16 18.3074V11.3574H17.5V19.2266C17.5 19.5619 17.3644 19.8172 17.0932 19.9925C16.8221 20.1678 16.5358 20.192 16.2346 20.065L11 17.8189L5.76537 20.065ZM16.7496 9.3574C16.537 9.3574 16.3589 9.28552 16.2154 9.14177C16.0718 8.99802 16 8.81991 16 8.60742V7.3574H14.75C14.5375 7.3574 14.3594 7.28549 14.2156 7.14167C14.0718 6.99787 14 6.81967 14 6.60707C14 6.39449 14.0718 6.21641 14.2156 6.07282C14.3594 5.92922 14.5375 5.85742 14.75 5.85742H16V4.60742C16 4.39492 16.0719 4.2168 16.2157 4.07305C16.3595 3.9293 16.5377 3.85742 16.7503 3.85742C16.9629 3.85742 17.141 3.9293 17.2845 4.07305C17.4282 4.2168 17.5 4.39492 17.5 4.60742V5.85742H18.75C18.9625 5.85742 19.1406 5.92933 19.2843 6.07315C19.4281 6.21696 19.5 6.39516 19.5 6.60775C19.5 6.82035 19.4281 6.99844 19.2843 7.14202C19.1406 7.28561 18.9625 7.3574 18.75 7.3574H17.5V8.60742C17.5 8.81991 17.428 8.99802 17.2842 9.14177C17.1404 9.28552 16.9622 9.3574 16.7496 9.3574Z"
      fill="currentColor"
    />
  </svg>
</div>
`;
